src/ozw/zwNode.cxx

```diff
@@ -187,8 +187,8 @@ void zwNode::dumpNode(bool all)
                   valueUnits.c_str(),
                   perms.c_str());
 
-          fprintf(stderr, "\t\t VID: %016" PRIx64 "\n",
-                  vid.GetId());
+          fprintf(stderr, "\t\t VID: %016" PRIu64 "\n",
+                  static_cast<uint64_t>(vid.GetId()));
         }
```

Comment:
This should be left as PRIx64 - a hexidecimal representation if the valueId's... This is much more useful than converting it into an unsigned int, since it represents a bitfield.  Also, is that cast necessary?